core/scripts/local/RequestOraclePrice.js

```diff
@@ -50,31 +59,20 @@ async function run(registryAddress, oracleAddress, identifier, timeInSeconds) {
 }
 
 const runRequestOraclePrice = async function(callback) {
-  // Usage: truffle exec scripts/RequestOraclePrice.js <Registry address> <CentralizedOracle address> <identifier> <time> --network <network>
+  // Usage: truffle exec scripts/RequestOraclePrice.js <identifier> <time> --network <network>
```

Comment:
nit: I think the usage should be `... --identifier <identifier> --time <time>...` since the args are now named.

Reply:
Done.